src/checkpoint/graphql/resolvers.ts

```diff
@@ -22,11 +22,15 @@ export async function queryMulti(parent, args, context: ResolverContext, info) {
   }
   const first = args?.first || 1000;
   const skip = args?.skip || 0;
-  const orderBy = 'created';
-  const orderDirection = 'DESC';
+
+  let orderBySql = '';
+  if (args.orderBy) {
+    orderBySql = `ORDER BY ${args.orderBy} ${args.orderDirection || 'DESC'}`;
+  }
```

Comment:
BREAKING CHANGE ALERT: This could potentially be a breaking change if the UI currently assumes that all muti-entity queries are implicitly ordered by the `created` column. 

We will have to update those when this gets merged.